metals/src/main/scala/scala/meta/internal/metals/watcher/FileWatcher.scala

```diff
@@ -111,7 +110,6 @@ object FileWatcher {
     buildTargets.sourceItems.foreach(collect)
 
     buildTargets.allTargetRoots
-      .filterNot(_.isJar)
```

Comment:
This jar target root will be the initial one which we resolve in `MetalsEntichments` ? If so should we resolve it here?